common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -733,16 +734,28 @@ def test_course_successors(self):
         self.assertEqual(len(result.children[0].children), 1)
         self.assertEqual(result.children[0].children[0].locator.version_guid, versions[0])
 
+    def _get_course_version(self, course_key, ms):
+        course_index = ms.get_course_index(course_key)
+        versions = course_index[u'versions']
+        return versions.get(ModuleStoreEnum.BranchName.published) or versions.get(ModuleStoreEnum.BranchName.draft)
+
+    def test_get_course_structure(self):
+        """
+        Tests the implementation of ModuleStoreRead.get_course_structure.
+        """
+        course_key = CourseLocator(org='testx', course='wonderful', run="run", branch=BRANCH_NAME_PUBLISHED)
+        self.assertValidCourseStructure(course_key, modulestore())
```

Comment:
Oh, nevermind. You've already covered that.

You should make sure that the course that you're testing against has inheritable data, though.